plugins/block-extension-tooltip/README.md

```diff
@@ -0,0 +1,63 @@
+# @blockly/block-extension-tooltip [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
+
+A [Blockly](https://www.npmjs.com/package/blockly) block extension that adds
```

Comment:
Add notes that:
- this is monkeypatching (until we make the correct change in core)
- this uses a block extension (link to the documentation)
- as a result, it has to be added to each block type